src/helix-ui/elements/HXElement.less

```diff
@@ -1,12 +1,11 @@
 @import (reference) "vars";
 
-*, *::before, *::after {
-  box-sizing: border-box;
-  color: inherit;
-  font: inherit;
-  letter-spacing: inherit;
-}
-
-input::-ms-clear {
-  display: none;
+:host {
```

Comment:
This is needed to eliminate global `*` definitions from being generated by ShadyCSS, instead it'll generate something like 

```html
<style scope="hx-search">
  *.hx-search { /* these defs only apply to <hx-search> ShadyDOM */ }
</style>
```
instead of 
```html
<style scope="hx-search">
  * { /* these defs will apply everywhere */ }
</style>
```